src/OrleansProviders/Streams/Memory/IMemoryStreamQueueGrain.cs

```diff
@@ -1,26 +1,22 @@
-﻿using System.Collections.Generic;
+﻿
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Orleans.Providers.Streams.Memory
+namespace Orleans.Providers
```

Comment:
What's the thinking of removing .Stream.Memory? GeneratorStreamProvider has .Stream.Generator in its namespace.